src/runtime/d3d12compute.cpp

```diff
@@ -1897,6 +1839,31 @@ static void *buffer_contents(d3d12_buffer *buffer) {
             pData = map_buffer(&readback);
             break;
 
+        /*
+        // TODO(marcos): this is probably how these buffer types should be handled:
```

Comment:
Why aren't they handled this way?

Reply:
yeah, this needs better explanation about why not

Reply:
I enabled those cases, but they will never get called in the current back-end implementation. I protected with a (paranoid) assert, just in case.